docs/content/configuration/index.md

```diff
@@ -257,6 +257,14 @@ This config is used to find the [Indexing Service](../design/indexing-service.ht
 |--------|-----------|-------|
 |`druid.selectors.indexing.serviceName`|The druid.service name of the indexing service Overlord node. To start the Overlord with a different name, set it with this property. |druid/overlord|
 
+### Broker Discovery
+
+This config is used to find the [Brokers](../design/broker.html) using Curator service discovery.
+
+|Property|Description|Default|
+|--------|-----------|-------|
+|`druid.selectors.broker.serviceName`|The druid.service name of the broker node. |druid/broker|
```

Comment:
2) actually keeping `druid.selectors.broker.serviceName` is OK as this is intended for coordinator to proxy requests to brokers (originally coming from something like UI). Are there use cases where we would want to proxy requests from coordinator to router instead?